### ISSUE TITLE
error: Attribute target redefined in SVG

### ISSUE THREAD

**Issue by jhasse** [doxygen/doxygen#8053]
Doxygen generates in invalid SVG in some cases (couldn't figure out exactly when), see https://bixense.com/jngl/classjngl_1_1Controller__inherit__graph.svg for example (here's my Doxyfile btw: https://github.com/jhasse/jngl/blob/master/doc/Doxyfile). Chrome shows the following error message:

```
error on line 24 at column 148: Attribute target redefined
```

It worked in an older Doxygen version (1.8.15) so I bisected it: cd581388f3d013c501e3cefbaf3e81cf93d46fcb is the first bad commit. If I revert that commit on master it works again.

Maybe related: #7706

**Comment by albert-github** [comment]
You posted some links to some external sources, these are non persistent so please:
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.  
- Please also specify the full doxygen version used (`doxygen -v`).

I also remember that with older versions there were some problems with double `target` tags and this was fixed, so your problem might have been fixed already (I don't know which doxygen version you run) and also when I look at the given github repository (https://github.com/jhasse/jngl.git) and run doxyen I get the message:
```
error: Tag file 'cppreference-doxygen-web.tag.xml' does not exist or is not a file.
```

where can I get this file ?

**Comment by albert-github** [comment]
From investigating the links you gave (and the underlying HTML source I saw you used doxygen 1.8.20).
In the mean time I've found the file: `cppreference-doxygen-web.tag.xml` and I'm able to reproduce the problem.

I don't think that the problem is related to #7706

**Comment by jhasse** [comment]
Thanks for investigating!

Yeah, I should try to create a minimal reproducible example but didn't had the time to come up with one.

As for the Doxygen version: I bisected using the Doxygen Git repository between Release_1_8_15 and master (55e15c8 at the time). cd58138 was the first bad commit, so 1.8.19 should be the first version where this is happening.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8058

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
